pkgs/development/tools/ocaml/camlp5/default.nix

```diff
@@ -4,12 +4,13 @@ if lib.versionOlder ocaml.version "4.02"
 then throw "camlp5 is not available for OCaml ${ocaml.version}"
 else
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
 
-  name = "camlp5-7.14";
+  pname = "camlp5";
+  version = "7.14";
 
   src = fetchzip {
```

Comment:
This can be replaced by `fetchFromGitHub`.

Reply:
@SuperSandro2000 suggested:

https://github.com/NixOS/nixpkgs/pull/126894#discussion_r651379506